drracket/drracket/private/drracket-normal.rkt

```diff
@@ -298,23 +300,24 @@
                                 (get-todays-icon)))
 
 (when (getenv "PLTDRBREAK")
-  (printf "PLTDRBREAK: creating break frame\n") (flush-output)
-  (let ([to-break (eventspace-handler-thread (current-eventspace))])
-    (parameterize ([current-eventspace (make-eventspace)])
-      (let* ([f (new frame% (label "Break DrRacket"))]
-             [b (new button% 
-                     (label "Break Main Thread")
-                     (callback
-                      (λ (x y)
-                        (break-thread to-break)))
-                     (parent f))]
-             [b (new button% 
-                     (label "Break All Threads")
-                     (callback
-                      (λ (x y)
-                        ((dynamic-require 'drracket/private/key 'break-threads))))
-                     (parent f))])
-        (send f show #t)))))
+  (printf "PLTDRBREAK: creating break frame\n")
+  (flush-output)
+  (define to-break (eventspace-handler-thread (current-eventspace)))
+  (parameterize ([current-eventspace (make-eventspace)])
+    (let* ([f (new frame% (label "Break DrRacket"))]
+           [b (new button% 
+                   (label "Break Main Thread")
+                   (callback
+                    (λ (x y)
+                      (break-thread to-break)))
+                   (parent f))]
+           [b (new button% 
+                   (label "Break All Threads")
+                   (callback
+                    (λ (x y)
+                      ((dynamic-require 'drracket/private/key 'break-threads))))
+                   (parent f))])
+      (send f show #t))))
```

Comment:
This could be turned into a `define` for `f` and then two things that just don't bind the buttons because `b` isn't actually referenced?

Reply:
I noticed that one too. Resyntax doesn't currently detect this case, but I opened https://github.com/jackfirth/resyntax/issues/185 for the feature request.